skills/nuxt-ui/references/layouts/chat.md

```
@@ -44,6 +35,35 @@ export default defineEventHandler(async (event) => {
+
+  return streamText({
+    model: openai('gpt-4o'),
+    system: 'You are a helpful assistant.',
+    messages: await convertToModelMessages(messages)
+  }).toUIMessageStreamResponse()
+})
+```
+
+## Component tree
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)